### PR TITLE
Add banner option in settings

### PR DESCRIPTION
**Pull request by JulienParis**
adding very little settings to allow banner display...

PR parallel to : https://github.com/etalab/udata-gouvfr/pull/480

- [x] to be cleaned from comments
- [ ] add banner html example ? 
- [ ] try banner on default theme ? 
- [ ] add test

### CODE REVIEW

**Comment by JulienParis** [comment]
Just needed to inject the banner settings to the context, either on homepage or others to make the banner visible on every page...

**Comment by JulienParis** [comment]
Capture d’écran 2020-06-11 à 08 10 39

**Comment by JulienParis** [comment]
Capture d’écran 2020-06-11 à 08 10 49

File: udata/core/site/views.py [hunk 1]
```diff
@@ -96,6 +96,8 @@ def home():
     }
     processor = theme.current.get_processor('home')
     context = processor(context)
+    context['banner_activated'] = current_app.config['BANNER_ACTIVATED']
```
Comment (abulte) [inline udata/core/site/views.py:99]: I'm not sure you need this. Did you try to using global `config` context var? It should be available in the templates, cf https://github.com/opendatateam/udata/blob/master/udata/templates/apidoc.html#L5 for example.

File: udata/frontend/views.py [hunk 2]
```diff
@@ -1,22 +1,28 @@
-from flask import request, redirect, abort, g, json
+import logging
+
+from flask import request, redirect, abort, g, json, current_app
 from flask.views import MethodView
 
 from udata import search, auth, theme
 from udata.utils import not_none_dict
 
+log = logging.getLogger(__name__)
```
Comment (abulte) [inline udata/frontend/views.py:9]: remove (plus the import) since you're not using it.

File: udata/frontend/views.py [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
 
 class Templated(object):
     template_name = None
 
     def get_context(self):
-        return {}
+        basic_context = {}
+        return basic_context
```
Comment (abulte) [inline udata/frontend/views.py:16]: Useless diff, revert to previous.

File: udata/core/site/views.py (hunk 1 above)
Comment (JulienParis) [reply udata/core/site/views.py]: changed that and [adapted the gouvfr theme for it](https://github.com/etalab/udata-gouvfr/pull/480/files#diff-60ca9eea7a34ed3b3dba3f52f6d46227), thx for the tip 👍

File: udata/settings.py [hunk 4]
```diff
@@ -439,6 +439,9 @@ class Defaults(object):
 
     API_DOC_EXTERNAL_LINK = 'https://doc.data.gouv.fr/api/reference/'
 
+    # Global banner parameters
+    BANNER_ACTIVATED = True
```
Comment (abulte) [inline udata/settings.py:443]: set it to `False` by default

File: udata/settings.py [hunk 5]
```diff
@@ -439,6 +439,9 @@ class Defaults(object):
...
+    BANNER_HTML_FILENAME = 'banner-2020-06-datactivist-survey.html'
```
Comment (abulte) [inline udata/settings.py:444]: set it to `''` by default

**Pull request merged**